types/pipeline_status.go

```diff
@@ -2,11 +2,27 @@ package types
 
 import "time"
 
+type PipelineEventSystem string
+
+const (
+	PipelineEventSystemDeployment PipelineEventSystem = "k8s:deployment"
+	PipelineEventSystemPod        PipelineEventSystem = "k8s:pod"
```

Comment:
I think EventSource makes it clear.

Reply:
No, they are not. I just noticed they both exist as well.

Reply:
Then I would rename back to `System` to avoid confusion.

Reply:
comments @niedbalski ?

Reply:
I think Source is okay, the Event.Source.Component should be used here, which should be translatable into our PIpelineEventSource type?

Reply:
I wanted to leave the 'k8s:' prefix in case we add other source layers, specifically if we later add fluent-bit logs straight from the container itself.